ivi-layermanagement-examples/simple-weston-client/CMakeLists.txt

```diff
@@ -107,6 +113,10 @@ add_executable(${PROJECT_NAME} ${SRC_FILES} ${WESTON_DEBUG_SRC_FILES})
 
 add_dependencies(${PROJECT_NAME} ${LIBS})
 
+if(${LIBWESTON_PROTOCOLS_FOUND})
+    add_definitions(${DLT_CFLAGS})
+endif(${LIBWESTON_PROTOCOLS_FOUND})
```

Comment:
I think you dont need this if defs. DLT_CFLAGS would be empty if dlt is not found